src/test/resources/emails/sessionClosedEmailForStudent.html

```diff
@@ -1,16 +1,14 @@
-Hello student1 In Course1</td></div>'",
+<p>Hello student1 In Course1</td></div>'",</p>
```

Comment:
Adding on to this review just to clarify a few things:

1. The `</td></div>'"` here is part of the course name used for testing, and is supposed to look like this. This should not be changed.
2. Usually there is no need to comment on every instance of a problem in the test HTML files, as these files are changed via [GodMode](https://github.com/TEAMMATES/teammates/blob/master/docs/godmode.md), and should not be changed directly. Therefore, repeated instances of the same issue in the test files are likely caused by the same piece of code that is being tested, and so adding a single comment to the original code is preferred (so there's no need to point out `</p>p>` in the test HTML files in this case, as they likely originate from the ones found in the main code)